src/Core/src/Attributes/PortTestAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿#nullable enable
+using System;
+
+namespace Microsoft.Maui
+{
+	[AttributeUsage(AttributeTargets.All, AllowMultiple = true)]
+	public class PortTestAttribute : Attribute
```

Comment:
Can we use the `MovedToAppium` attribute? That one also inherits from Ignore which is useful because we don't want to run this in the compatibility runner anymore